src/test/java/subway/line/LineAcceptanceTest.java

```diff
@@ -25,31 +27,35 @@ public class LineAcceptanceTest {
 
     private Map<String, String> newBunDangLineParams;
     private Map<String, String> zeroLineParams;
-    private final StationApiCaller stationApiCaller = new StationApiCaller();
-    private final LineApiCaller lineApiCaller = new LineApiCaller();
+    private Long firstStationId;
+    private Long secondStationId;
+    private Long thirdStationId;
+    private Long forthStationId;
 
     @BeforeEach
     void setUpClass() {
         Map<String, String> params = new HashMap<>();
         params.put("name", "강남역");
-        Long firstId = stationApiCaller.callCreateStation(params).jsonPath().getObject("id", Long.class);
+        firstStationId = StationApiCaller.callCreateStation(params).jsonPath().getObject("id", Long.class);
         params.put("name", "삼성역");
-        Long secondId = stationApiCaller.callCreateStation(params).jsonPath().getObject("id", Long.class);
+        secondStationId = StationApiCaller.callCreateStation(params).jsonPath().getObject("id", Long.class);
         params.put("name", "선릉역");
-        Long thirdId = stationApiCaller.callCreateStation(params).jsonPath().getObject("id", Long.class);
+        thirdStationId = StationApiCaller.callCreateStation(params).jsonPath().getObject("id", Long.class);
+        params.put("name", "교대역");
+        forthStationId = StationApiCaller.callCreateStation(params).jsonPath().getObject("id", Long.class);
```

Comment:
코드를 읽다 보면 n번째 역 ID가 어떤 역이었는지 파악하기 어려울 경우가 있더라구요
인수 테스트의 가독성을 위해 한글 변수명 사용은 어떠신가요?
jsonPath를 활용해 id를 추출하는 코드도 분리한다면 테스트 가독성에 도움이 될 것 같아요 
```suggestion
        params.put("name", "교대역");
        교대역_ID = StationApiCaller.callCreateStation(params);
```